content/articles/creating-file-management-application-using-winforms/index.md

```diff
@@ -0,0 +1,241 @@
+
+We shall demonstrate how you can create files and folders, write data to them in your interested formats, read the memory data, delete the initially recorded files, and view all data using WinForms.
+
+### Prerequisites
+1. Basic c# programming language and .NET framework.
+2. A machine with [Visual Studio](https://visualstudio.microsoft.com/vs/community/) running.
+
+### Table of contents
+- [Prerequisites](#prerequisites).
+- [Visual Studio setup](#step-1-visual-studio-setup).
+- [Designing the WinForm](#step-2-designing-the-winform).
+- [Including the necessary namespace](#including-the-necessary-namespace).
+- [Writting the C# code](#step-3-writting-c#-code)
+- [Debugging the application](#step-4-debugging-the-application).
+- [Conclusion](#conclusion).
+
+Let's get started!!
+
+### Step 1. Visual studio setup
+- Open visual studio on your device.
+- Select *`create new project`* in the Get Started window.
+
+![Creating a new project](engineering-education/creating-file-management-application-using-winforms/create-a-new-project.jpg)
+
+- Click *Visual C#* and select *` Windows Desktop`* from its rows data.
+- On the right side of the window, select *`Windows Forms App (.Net Framework)`* as the default template for this project.
+- On the bottom of the window, rename the project solution to *File manager*.
+- You can change the project location by clicking the browse button and choosing where the project will be saved. Otherwise, let it remain in its default.
+- Click *`OK`* to finish setting up the project solution.
+
+![Configuring the project settings](engineering-education/creating-file-management-application-using-winforms/configuring-the-project.jpg)
+
+### Step 2. Designing the WinForm
+Here we shall be modifying the form by adding controls such as labels, buttons, e.t.c., which will help the user interact with the application.
+
+#### Modifying the form name
+First, we need to rename the form from *Form 1* to *File manager* by changing its text property, as shown in the image below.
+
+The start position property of the form is optional to modify since it just sets its position on the device screen during its run-time, which is not that important.
+
+![Renaming the form application](engineering-education/creating-file-management-application-using-WinForms/changing-the-name-of-the-form.jpg)
+
+#### Using controls to construct the form
+We shall be using common tabs in our project windows such as `ToolBox` and the `Properties` to modify the form structure.
+
+To access the ToolBox, on the upper row of the Visual Studio window, click *` View -> ToolBox`*, and the tab will pop up on the left side of the main window.
+
+To access the properties window, click *` View -> Properties`* or right-click on the control you need to change its properties in the form, in the dialogue box that will appear choose property.
+
+#### 1. File browsing controls
```

Comment:
Please organize the flow here, it's too difficult to read along. Can you organize it into headings or subsections for the reader to understand.